gslb/nodes/avi_model_nodes.go

```diff
@@ -155,7 +155,7 @@ func (hm HealthMonitor) GetHMDescription(gsName string, template *string) []stri
 	descPrefix := CreatedByAMKO + ", gsname: " + gsName
 	desc = append(desc, descPrefix)
 	if hm.Type == NonPathHM {
-		if hm.Name == gslbutils.SystemGslbHealthMonitorPassthrough {
+		if hm.Name == gslbutils.SystemGslbHealthMonitorPassthrough+gslbutils.AMKOControlConfig().GetAMKOUUID() {
```

Comment:
What will be the impact on AMKOUUID change?

Reply:
Did we test upgrade scenario?

Reply:
on amkouuid change new hm will be created with name including the new amkouuid in name. 
Tested the upgrade scenario where gs was created with  amko--passthrough-hm-tcp  as hm attached. on upgrade new hm with new name was attached to GS.